nhsuk-design-system/views/includes/header-v1/site-layout.html

```diff
@@ -21,7 +21,8 @@
     <link rel="stylesheet" href="/css/prism.css">
 
     <script src="/js/clipboard.min.js"></script>
-
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script> 
```

Comment:
When including externally hosted resources make sure you include Subresource Integrity checking.

> The integrity and crossorigin attributes are used for Subresource Integrity (SRI) checking. This allows browsers to ensure that resources hosted on third-party servers have not been tampered with. Use of SRI is recommended as a best-practice, whenever libraries are loaded from a third-party source. Read more at srihash.org

Example:

`<script
  src="https://code.jquery.com/jquery-3.3.1.min.js"
  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
  crossorigin="anonymous"></script>`